tools/Nuget/template.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I've used Paket templating in one of my projects - [here](https://github.com/Horusiath/Akkling/blob/master/src/Akkling/Akkling.fsproj.paket.template) is the example paket.template file, and [here](https://github.com/Horusiath/Akkling/blob/master/build.fsx#L144) you can see, how to use Paket commands from within the FAKE build scripts.